examples/kernel_shap_wine_intro.ipynb

```diff
@@ -0,0 +1,1201 @@
+{
```

Comment:
<p>Since the user has to do the selection themselves, should we reword this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/192/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>